ansible/roles/monitoring-client/templates/collectd/collectd.client.conf.j2

```diff
@@ -96,6 +95,12 @@ LoadPlugin "logfile"
   Import NginxHls
   Import NginxDash
 {% endif %}
+{% if transcoder_streams is defined or transcoding_lounge is defined %}
+  Import "TranscodingUnits"
+  <Module TranscodingUnits>
+    UnitRegex "transcode_(h264|vpx|audio)@[a-zA-Z0-9]{2,2}\.service" 
```

Comment:
Will. this match other encoders like MUCC etc?

Reply:
ohh you're right, i should fix this.